src/actions/create.js

```diff
@@ -30,31 +33,33 @@ import { extendMetaWithResponse, buildRSAAConfig } from '../rsaa';
  * @returns {function}
  */
 export function create(schema, item = null, params = {}, options = {}) {
-  const config = resolveConfig(schema);
+  const config = resolveResourceConfig(schema, 'create');
   if (!config) {
-    const schemaName = schema && _.isObject(schema) ? schema.schema : schema;
+    const schemaName = schema && _.isObject(schema) ? resolveSchemaType(config) : schema;
     throw new Error(`Couldn't resolve schema ${schemaName} in function find.`);
   }
 
   const rsaaConfig = buildRSAAConfig(config);
   const endpoint = buildEndpoint(rsaaConfig.endpoint, params, options);
+  const source = _.get(config, 'request.resourceType', JSON_API_SOURCE);
```

Comment:
Can we use `JSON` as the default source?

Reply:
For now I would leave json-api because it is only supported format. Once we add JSON support I would refactor this part.